app/models/topic.rb

```diff
@@ -1307,8 +1307,8 @@ def self.with_no_response_total(opts = {})
     builder.query_single.first.to_i
   end
 
-  def convert_to_public_topic(user)
-    public_topic = TopicConverter.new(self, user).convert_to_public_topic
+  def convert_to_public_topic(user, opts = nil)
```

Comment:
Is it an opts object? Looks like it's only supporting one optional parameter: category_id.

Reply:
Good catch, fixed.